docs/monitoring-usage.rst

```diff
@@ -278,3 +278,110 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Time Series
+---------------------------
+
+The Stackdriver Monitoring API can be used to create and delete custom metrics, as well as write
+data points to them. Please refer to the documentation for the `Creating Custom Metrics`_
+documentation for more information.
+
+
+To create a custom TimeSeries value, you must first create a custom
+:class:`~gcloud.monitoring.metric.MetricDescriptor` object, and then use its
+type to create a fully parametrized :class:`~gcloud.monitoring.metric.Metric` by providing specific
+values for the available labels. You must create a fully parametrized  :class:`~gcloud.monitoring.resource.Resource` from one of the
+available ResourceDescriptors which can be listed using the :class:`~gcloud.monitoring.resource.ResourceDescriptor` class,
+which similarly requires that its available labels have values specified.
+
+>>> from gcloud import monitoring
+>>> import datetime
+>>> client = monitoring.Client()
+>>> label = LabelDescriptor('response_code', LabelValueType.INT64,
+...    description='HTTP status code')
+>>> metric_descriptor = client.metric_descriptor(
+...    'custom.googleapis.com/my_app/response_count',
+...    metric_kind=MetricKind.CUMULATIVE,
+...    value_type=ValueType.INT64,
+...    labels=[label],
+...    description='Cumulative count of HTTP responses.')
+>>> if not metric_descriptor.exists():
+>>>     metric_descriptor.create()
+>>> # Create a Resource by its resource name, assign labels as dict
+>>> resource = client.resource('gce_instance',
+...    labels={
+...    'instance_id': 'my-instance',
+...    'zone': 'us-central1-f'
+...    })
+>>> # Create a Metric using its metric descriptor parametrized with labels
+>>> metric = client.metric(descriptor=metric_descriptor, labels={
+...    'response_code': 404
+... })
+
+It's also possible to assign labels to Metrics and Resources with keyword arguments::
```

Comment:
This enhancement of the `Metric` and `Resource` classes can be a separate pull request.